aspnetcore/mvc/views/tag-helpers/builtin/resources/CacheTagHelper.md

```diff
@@ -0,0 +1,278 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+
+# CacheTagHelper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+
+The `CacheTagHelper` provides the ability to dramatically improve the performance of your ASP.NET core app by caching its content to the internal ASP.NET core cache provider.
+
+A simple example that shows the `CacheTagHelper` in action sets the current time in the content area of the Cache Tag Helper. The Razor View Engine sets the default of `expires-after` to be 20 minutes (this is the default if you specify no addional attributes).
+
+Here is an example of the code you would include in your `cshtml` page to achieve the default cache behavior described above.
+
+```HTML
```

Comment:
Lowercase to "html" here ... It will work either way, but for consistency across docs.